packages/@lwc/compiler/package.json

```diff
@@ -16,7 +16,8 @@
     "license": "MIT",
     "scripts": {
         "clean": "rm -rf dist/",
-        "build": "tsc && node ./scripts/update-compiler-version.js"
+        "build": "tsc && node ./scripts/update-compiler-version.js",
```

Comment:
What are your thoughts about making `build` use the `incremental` flag?

Reply:
I think we have two options:

1. Use `incremental` only for `yarn dev`. This will prevent `tsconfig.tsbuildinfo` files from getting into `dist` directories when we publish (which should only happen on CI, which doesn't run `yarn dev`).
2. Use `incremental` everywhere. This will make `yarn build` faster locally, but we will need to configure `tsc` to write its `tsconfig.tsbuildinfo` file somewhere else so it doesn't end up in `dist`.

Reply:
I don't have a strong opinion on the 2 options. This PR already gives a big productivity boost in dev. I am fine not using `--incremental` for `build`.